lib/routes/sycltech/feeds.ts

```diff
@@ -0,0 +1,47 @@
+import { Route } from '@/types';
+import ofetch from '@/utils/ofetch'; // 统一使用的请求库
+import { parseDate } from '@/utils/parse-date';
+
+export const route: Route = {
+    path: '/:feed?',
+    categories: ['programming'],
+    example: '/sycltech/news',
+    parameters: { feed: 'feed来源，默认为news，参考https://feeds.sycl.tech/' },
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    name: 'Feeds',
+    maintainers: ['mocusez'],
+    handler,
+    description: `|  活动  | 新闻 |    研究文章     |  视频  |
+    | :----: | :--: | :-------------: | :----: |
+    | events | news | research_papers | videos |`,
```

Comment:
```suggestion
| :----: | :--: | :-------------: | :----: |
| events | news | research_papers | videos |`,
```
Indent the description with either 0 or 2 spaces, otherwise the markdown won't be rendered properly.